media_test_harness/runtests.py

```diff
@@ -6,21 +6,65 @@
 import os
 import sys
 
-from firefox_puppeteer import manifest as puppeteer_manifest
-from firefox_ui_harness import cli
-from firefox_ui_harness.options import FirefoxUIOptions
-from firefox_ui_harness.runners import FirefoxUITestRunner
-from firefox_ui_tests import manifest_all as ui_manifest
-from firefox_ui_tests import resources as ui_resources
+import mozlog
+from marionette import BaseMarionetteTestRunner, BaseMarionetteOptions
+from marionette.marionette_test import MarionetteTestCase
 
 import firefox_media_tests
 from testcase import MediaTestCase
 
-
-class MediaTestOptions(FirefoxUIOptions):
+DEFAULT_PREFS = {
+    'app.update.auto': False,
+    'app.update.enabled': False,
+    'browser.dom.window.dump.enabled': True,
+    # Bug 1145668 - Has to be reverted to about:blank once Marionette
+    # can correctly handle error pages
+    'browser.newtab.url': 'about:newtab',
+    'browser.newtabpage.enabled': False,
+    'browser.reader.detectedFirstArticle': True,
+    'browser.safebrowsing.enabled': False,
+    'browser.safebrowsing.malware.enabled': False,
+    'browser.search.update': False,
+    'browser.sessionstore.resume_from_crash': False,
+    'browser.shell.checkDefaultBrowser': False,
+    'browser.startup.page': 0,
+    'browser.tabs.animate': False,
+    'browser.tabs.warnOnClose': False,
```

Comment:
As far as e10s goes, I think we should test whatever is the default on the given channel. e10s will be on by default on Nightly, so there's no need to set a pref here, and I will add an --e10s-off option below.